content/cn/docs/guides/toolchain-local-test.md

```
@@ -0,0 +1,523 @@
+
+### 3.1 使用脚本快速部署测试环境（推荐）
+
+这种方式允许您从源代码编译和安装特定版本的 HugeGraph Server，确保测试环境与特定 HugeGraph Server 版本的一致性，这对于复现问题或验证兼容性至关重要。
+
+#### 3.1.1 变量与参数
+
+*   **`$COMMIT_ID`**
+    *   指定 HugeGraph Server 源代码的 Git Commit ID。当您需要从源代码编译和安装特定版本的 HugeGraph Server 作为测试依赖时，会使用此变量,确保测试环境与特定 HugeGraph Server 版本的一致性，这对于复现问题或验证兼容性至关重要。使用时直接接作为参数传递给 install-hugegraph-from-source.sh 脚本。
+
+*   **`$DB_DATABASE` 与 `$DB_PASS`**
+    *   指定 HugeGraph-Loader 进行 JDBC 测试时所连接的 MySQL 数据库的名称和root 用户密码。提供数据库连接信息，使 Loader 能够正确地读写数据。使用时直接接作为参数传递给 使用时直接接作为参数传递给 install-mysql.sh 脚本。
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)